### PR TITLE
Location id changes for fe

### PR DESCRIPTION
**Pull request by mksrivastava**

### CODE REVIEW

File: src/app/core/services/establishment.service.ts [hunk 1]
```diff
@@ -49,6 +49,19 @@ interface MainServiceRequest {
     other?: string;
   };
 }
+interface CQCDatilsRequest {
```
Comment (aaron-russell) [inline src/app/core/services/establishment.service.ts]: Should this be CQC Details?

File: src/app/core/services/establishment.service.ts [hunk 2]
```diff
@@ -49,6 +49,19 @@ interface MainServiceRequest {
...
+  addressLine1: string;
+  addressLine2: string;
```
Comment (aaron-russell) [inline src/app/core/services/establishment.service.ts]: Address Line 2 can be optional too

File: src/app/features/workplace/select-workplace/select-workplace.component.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
+<app-error-summary *ngIf="submitted && form.invalid" [formErrorsMap]="formErrorsMap" [form]="form"> </app-error-summary>
```
Comment (aaron-russell) [inline src/app/features/workplace/select-workplace/select-workplace.component.html]: Could these not use the existing templates?

File: src/app/core/services/establishment.service.ts (hunk 1 above)
Comment (mksrivastava) [reply src/app/core/services/establishment.service.ts]: ok change it to CQCLocationChangeRequest

File: src/app/core/services/establishment.service.ts (hunk 2 above)
Comment (mksrivastava) [reply src/app/core/services/establishment.service.ts]: made it optional and in other models as well.

File: src/app/features/workplace/select-workplace/select-workplace.component.html (hunk 3 above)
Comment (mksrivastava) [reply src/app/features/workplace/select-workplace/select-workplace.component.html]: yes made the changes for common template

**Pull request merged**